pymc3/distributions/multivariate.py

```diff
@@ -473,28 +473,31 @@ class Multinomial(Discrete):
     Parameters
     ----------
     n : int or array
-        Number of trials (n > 0).
+        Number of trials (n > 0). If n is an array its shape must be (N,) with 
+        N = p.shape[0]
     p : one- or two-dimensional array
         Probability of each one of the different outcomes. Elements must
-        be non-negative and sum to 1 along the last axis. They will be automatically
-        rescaled otherwise.
+        be non-negative and sum to 1 along the last axis. They will be 
+        automatically rescaled otherwise.
     """
 
     def __init__(self, n, p, *args, **kwargs):
         super(Multinomial, self).__init__(*args, **kwargs)
 
         p = p / tt.sum(p, axis=-1, keepdims=True)
```

Comment:
unrelated to the PR, but does `p` need to be scaled here if it is also scaled in the `._random` method?

Reply:
I think so, just to make sure the logp is correct.
The rescaling in `._random` is more to dealt with the precision issue.